parsec/core/gui/main_window.py

```diff
@@ -271,11 +271,18 @@ def _on_finished(ret):
         )
 
     def _on_join_org_clicked(self):
+        default_url = ""
+        try:
+            # Get the clipboard text, try and convert it to parsec addr and use it as default value
+            default_url = BackendActionAddr.from_url(desktop.get_clipboard()).to_url()
+        except:
```

Comment:
Can you specify which kind of exception you want to catch here ? 

Maybe you don't need also to catch `BaseException` here and get only:
 `except Exception:`